tortoise/contrib/test/__init__.py

```diff
@@ -21,6 +21,56 @@
 On success it will be marked as unexpected success.
 """
 
+_CONFIG = {}  # type: dict
+_APPS = {}  # type: dict
+_CONNECTIONS = {}  # type: dict
+
+
+def getDBConfig(app_label: str, model_modules: List[str]) -> dict:
+    """
+    DB Config factory, for use in testing.
+    """
+    return _generate_config(
+        _TORTOISE_TEST_DB,
+        model_modules=model_modules,
+        testing=True,
+        app_label=app_label,
+    )
+
+
+async def _init_db(config):
+    try:
+        await Tortoise.init(config)
+        await Tortoise._drop_databases()
+    except DBConnectionError as exc:  # pragma: nocoverage
+        pass
+
+    await Tortoise.init(config, _create_db=True)
+    await Tortoise.generate_schemas()
+
+
+def initializer():
+    global _CONFIG
+    global _APPS
+    global _CONNECTIONS
+    _CONFIG = getDBConfig(
+        app_label='models',
+        model_modules=['tortoise.tests.testmodels'],
+    )
+
+    loop = _asyncio.get_event_loop()
+    loop.run_until_complete(_init_db(_CONFIG))
+    _APPS = Tortoise.apps.copy()
```

Comment:
May be deepcopy should be used here? `apps` defined with nested structure, so deepcopy should be safer in case of some test mutating apps structure

Reply:
Good point.